src/index.html

```diff
@@ -8,6 +8,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rate stars--0">
```

Comment:
BEM modifier(`stars--0`) should be used with the corresponding block
Why do you need an additional block rate?

Reply:
fixed